lib/tandem/tandemSimulator.js

```diff
@@ -165,6 +165,9 @@ exports.make = function(config){
     changeReservoir: function(event) {
       simpleSimulate(event);
     },
+    cartridge: function(event) {
```

Comment:
Ditto here with primes not really being cartridge-related. I think I personally might prefer a name like `miscDeviceEvent` or something instead.